cli/cmd/component-apply.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 var componentApplyCmd = &cobra.Command{
-	Use: "apply",
```

Comment:
(Commenting here because GH doesn't allow comments on the commit's description)

Typo in `Fix tens[e]` (also `ran` is not the future, but you don't even have to name the tenses :) ).

Reply:
Oops. Fixed the typo :-)

Ran is past tense, and "If ran" is wrong because the intention here is "if the command **is run**", i.e. "if someone runs the command" but in passive form.

The "future -> present" is another fix, e.g. "will apply" -> "applies".

Reply:
I've amended the commit message to make it clearer that I'm referring to two fixes.